mirage/scenarios/collections.ts

```diff
@@ -124,18 +134,116 @@ export function collectionAcceptedScenario(server: Server, currentUser: ModelIns
         id: nodeAdded.id,
         collection: primaryCollection,
     });
+    server.create('collection-provider', {
+        id: 'accepted-collection-example',
+        primaryCollection,
+        licensesAcceptable,
+    });
+}
+
+export function collectionRejectedScenario(server: Server, currentUser: ModelInstance<User>) {
+    const licensesAcceptable = server.schema.licenses.all().models;
+    const primaryCollection = server.create('collection');
+    const nodeToBeAdded = server.create('node', {
```

Comment:
The variable `nodeToBeAdded` is used but only to add a contributor to it.. The actual node that you're creating on this line isn't used for a collection-submission, so it's currently not needed. If you meant to use it for a collection-submission later on, it can stay, but as of right now, it's just a node in mirage that's not being used.